src/config/lang/english.go

```diff
@@ -77,10 +77,10 @@ const (
 	CmdConnectListShort = "Lists all available connection shortcuts"
 
 	CmdConnectFlagName       = "Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6."
-	CmdConnectFlagNamespace  = "Specify the namespace.  E.g. namespace=default. Ignored if --name is unset."
-	CmdConnectFlagType       = "Specify the resource type.  E.g. type=svc or type=pod. Ignored if --name is unset."
-	CmdConnectFlagLocalPort  = "(Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000. Ignored if --name is unset."
-	CmdConnectFlagRemotePort = "Specify the remote port of the resource to bind to.  E.g. remote-port=8080. Ignored if --name is unset."
```

Comment:
We should continue to ignore namespace, type, and remote port if the target is set as it will always either be set correctly or not found by Zarf and error.

Reply:
I'll update the text to say `Ignored if connect-name is specified.` for those flags.  I was unsure if we wanted to always allow the user to override what Zarf finds.  Thanks!

Reply:
Totally fair to be confused there, I believe two separate commands would provide a better, less confusing UX. I will write up an issue about this later, but don't worry about that for this PR.

Reply:
I suppose `Name` should also be ignored if a target is specified.